docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,365 @@
+{
```

Comment:
<p>This paragraph needs to be updated to reference the ISA of the Heron processors, which is <code>cz</code> based rather than <code>ecr</code>.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>